bot.py

```diff
@@ -60,26 +96,27 @@ def evaluate(code, nickname):
 
     for line in lines:
         if len(line) > 150:
-            return pastebin(code)
+            pastebin(channel, code)
 
     limit = 3
     if len(lines) > limit:
-        return "\n".join(lines[:limit - 1] + [pastebin(code)])
+        return "\n".join(lines[:limit - 1] + [pastebin(channel, code)])
 
     return out
 
 class RustEvalbot(irc.client.SimpleIRCClient):
-    def __init__(self, nickname, channels, keys):
+    def __init__(self, nickname, channels, keys, triggers):
         irc.client.SimpleIRCClient.__init__(self)
         irc.client.ServerConnection.buffer_class = irc.buffer.LenientDecodingLineBuffer
         self.nickname = nickname
         self.channels = channels
         self.keys = keys
+        self.triggers = [(re.compile(r), c, wt) for (r, c, wt) in triggers]
```

Comment:
I guess it could be, but I'm not a fan of arbritary string keys. 
Also, the configuration the irc.yaml might be shorter like this (though shorter != better).

Is it necessary?